### PR TITLE
Starter code

### PR DESCRIPTION
**Pull request by Madhuri97**

### CODE REVIEW

File: project1/NOTES.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+TASK-1: Postgresql
```
Comment (ChaitanyaPuritipati) [inline project1/NOTES.md:1]: Time elapsed for task-1 is missing. Probably in this task you may not have any questions or tricky steps or online references to include. But from the next task it is important to include all the steps i.e process of you have done, online references, questions you addressed during the task and tricky/consuming steps

File: project1/NOTES.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+1. Visited heroku website and created free account 
+2. Now clicked "Create app" gave name as "wp-project1-105"
+3. Then opened overview, clicked on "Configure Add-ons" 
+4. In Addons searched for "heroku Postgres" after loading clikc over "Provision".
+5. Now i had click on heroku Postgres link -- settings -- view credentials 
+6. open Adminer link: paste the heroku credentials in this site and don.
+
+TASK-2: Python and FLASK
```
Comment (ChaitanyaPuritipati) [inline project1/NOTES.md:11]: Time elapsed for task 2 is missing

File: project1/NOTES.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+
+1. I already had python in my laptop and also had pip installed 
+2. Downloaded the project1 zip file to new repo named project-1-105 and branch named starter-code
+3. Opened terminal at project1 file location "py -m pip install -r requirements.txt" packages are installed 
+4. Set environment by "set FLASK_APP=application.py" -- "set FLASK_DEBUG=1" -- "DATABASE_URL= pasted my postegres uri"
+5. Checked for version of Werkzeug if it is 1.0.1 we need to uninstall and reinstall 0.16.0 version according our system. 
+6. "py -m pip install Werkzeug==0.16.0"
+7. Now "flask run" and copy http link and test over internet
+
+TASK-3: Goodreads API
```
Comment (ChaitanyaPuritipati) [inline project1/NOTES.md:28]: Time elapsed for task 3 is missing

**Review by ChaitanyaPuritipati — CHANGES_REQUESTED** [review]

File: project1/NOTES.md (hunk 1 above)
Comment (Madhuri97) [reply project1/NOTES.md:1]: i have updated the task

File: project1/NOTES.md (hunk 2 above)
Comment (Madhuri97) [reply project1/NOTES.md:11]: i have updated task

File: project1/NOTES.md (hunk 3 above)
Comment (Madhuri97) [reply project1/NOTES.md:28]: i have updated task

**Review by ChaitanyaPuritipati — APPROVED** [review]
Good work. Now you can merge this branch with the master and start working

**Pull request merged**